web/js/components/about/about-section.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export default function AboutSection(props) {
+  const { section } = props;
+
+  const [sectionDescription, setSectionDescription] = useState();
+
+  const fetchHTML = (sectionName, setFn) => {
+    let controller = new AbortController();
+    (async () => {
+      if (!sectionName) {
+        setFn('no section name provided');
+        return;
+      }
+      try {
+        const data = await fetch(`brand/about/${sectionName}.html`);
+        const dataHTML = await data.text();
+        controller = null;
+        const setData = data.ok ? dataHTML : 'No description available for this section.';
```

Comment:
I tried updating the "Disclaimer" section and when I did it appears to erroneously show the "No description available for this section" message:
<img width="699" alt="Screen Shot 2022-01-25 at 12 06 05 PM" src="https://user-images.githubusercontent.com/1480000/151024504-281f8d55-0e20-4c5e-b16d-64697c3100f1.png">

Reply:
Addressed in https://github.com/nasa-gibs/worldview/pull/3798/commits/2d61241e85c5d4b24aabbca4a2b35cc16b86f0fe and https://github.com/nasa-gibs/worldview/pull/3798/commits/3e8c87c069af1cd80a805225f5338349653f7cd7

Reply:
Related to this... if someone didn't want to have all sections, or all the _same_ sections as we use, they would still have to either include something in the markdown file with that name or modify the JavaScript right? Otherwise they would get this "not found" message.  We don't have to make this changes for this PR but in the future it might make more sense to just read in all the markdown from the config folder and insert it in order (you'd perhaps need to prefix each markdown file with sequential numbers to get the order you want).

Alternatively, maybe it doesn't need to display a "not found" message?  Or just to the console?  Up to you.

Reply:
I like the numbering idea.  That would be good because it keeps the control of the about sections solely on the markdown files.

And yes, I agree that it doesn't need to display a "not found" message.  I'll remove that and push up the changes.